tools/wave/ecmascript/test-template.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<title>{{ TEST_TITLE }}</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<!-- <script charset="UTF-8" src="/ecmascript/harness/jquery-1.4.2.min.js"></script> -->
```

Comment:
removed commented-out code (I'm not sure this path to jQuery is valid if it was uncommented?)